.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -4,23 +4,23 @@
 
 ## Does this introduce a breaking change?
 <!-- Mark one with an "x". -->
-```
-[ ] Yes
-[ ] No
-```
+
+- [ ] Yes
+- [ ] No
+
 
 ## Pull Request Type
 What kind of change does this Pull Request introduce?
 
 <!-- Please check the one that applies to this PR using "x", and prefix your PR title with the corresponding `type` -->
-```
-[ ] Bugfix (`fix`)
-[ ] Feature (`feat`)
-[ ] Code style update (formatting, local variables) (`style`)
-[ ] Refactoring (no functional changes, no api changes) (`refactor`)
-[ ] Documentation content changes (`docs`)
-[ ] Other... Please describe: (`perf`, `test`, `build`, `ci`, `chore`, `revert`)
-```
+
+- [ ] Bugfix (`fix`)
```

Comment:
You could argue that this is superfluous now as you have to have this in the title. If we are changing this template - that's  amend the text to say that

Reply:
makes perfect sense. updating.